package.json

```diff
@@ -142,7 +142,7 @@
     "source-map-support": "^0.5.16",
     "strip-bom": "^2.0.0",
     "testcafe-browser-tools": "2.0.26",
-    "testcafe-hammerhead": "31.7.1",
+    "testcafe-hammerhead": "^31.7.2",
```

Comment:
I think we should keep the version fixed. We need to ensure that the new version of testcafe-hammerhead doesn't break anything. Otherwise, users might encounter issues if the package has problems.